lambdaclass/mirra_backend
# Add loadtest client chile

## Motivation

We need to loadtest the new chile servers.

## Summary of changes

[Add chile loadtest client workflow and chile loadtest env vars](https://github.com/lambdaclass/mirra_backend/pull/887/commits/a1251b68e56e9173726a617cdee903d13c6b27ab)

## How to test it?

I already deployed it before and worked.
You can SSH to the server and see it running. The deploy action is this one -> [Fix release for testing purposes](https://github.com/lambdaclass/mirra_backend/actions/runs/10600855196)

## Checklist
